### DISCUSSION TITLE
wxWidgets 3.1.6 , running application has missing dll "pcre2-16.dll"

### DISCUSSION THREAD

**Discussion by beemibrahim** [microsoft/vcpkg#24800] [Q&A]
When I run my wxWidgets 3.1.6 program I get this error : 

![Screenshot 2022-05-19 133550](https://user-images.githubusercontent.com/92852749/169250428-8abf4511-23a7-4686-848c-d7e8948e959e.png)

I installed wxWidgets 3.1.6 this : 

```
vcpkg install wxwidgets:x64-windows

vcpkg integrate install
```

I am using the latest vcpkg.

**Edit :** 

I checked if I had pcre2 installed and I did : 

```
vcpkg install pcre2:x64-windows
```

```
Computing installation plan...
The following packages are already installed:
    pcre2[core]:x64-windows -> 10.40
pcre2:x64-windows is already installed
Restored 0 packages from C:\Users\Ibrahim\AppData\Local\vcpkg\archives in 96.3 us. Use --debug to see more details.

Total elapsed time: 40.58 ms
```
And it seems like `pcre2-16.dll` should come with wxWidgets am I not right ?

**Edit :**

I searched the vcpkg folder and found the file `pcre2-16.dll` , I then copied the file to my application directory , and then It compiled an ran properly. Shouldn't vcpkg copy the dependency to the output folder itself ?

**Comment by USAFrenzy** [comment]
You may just need to update your vcpkg, your issue seems to be a duplicate of #21255 which was fixed by the merge from #21248

**Reply by USAFrenzy** [reply]
It's either that, or the version of wxWidgets may or may not be up-to-date yet in vcpkg - hopefully someone who knows more can help ya out but I would definitely try just updating vcpkg

**Reply by USAFrenzy** [reply]
Oh nevermind! stumbled across your other post while searching the discussions and it does seem it was a mismatch and that you have it working now - disregard my answer here.
